test/e2e/e2e-helpers.js

```diff
@@ -87,6 +87,81 @@ function overrideAnimations(client) {
   });
 }
 
+/**
+ * Set up some helper functions to make writing tests nicer
+ */
+function bootstrapHelpers(client) {
```

Comment:
Ideally, these boostrapped helpers would be in the `client`'s `prototype`, I think, but at this point, I think it's neither here nor there with the level of control we have.